regression/cbmc/Typecast2/main.i

```diff
@@ -0,0 +1,7 @@
+int main()
+{
+  unsigned int i=2;
+  __CPROVER_assert(0l==(signed long int)(i - (unsigned int)2),
+                   "difference of cast");
```

Comment:
Why is this change necessary and/or beneficial?

Reply:
When run on a 32-bit system, you would otherwise need a cross-compiler to 64-bit.

Reply:
Oh, of course, I'm sorry for having been dense. Yet it would be nice if such information were included in the commit message.

Reply:
Ok, now in commit message.